### ISSUE TITLE
Fix warnings from python 3.10 at the codebase

### ISSUE THREAD

**Issue by johnnv1** [kornia/kornia#2144]
After we ensured the support for python 3.10, and added it to CI we are seeing a bunch of warning. Just a good code health issue 😄 

<details><summary>Log / warnings</summary>

test/test_contrib.py::TestImageStitcher::test_exception[cpu-float32]
test/feature/test_loftr.py::TestLoFTR::test_mask[cpu]
test/feature/test_loftr.py::TestLoFTR::test_gradcheck[cpu]
  /home/runner/work/kornia/kornia/kornia/feature/loftr/utils/coarse_matching.py:243: UserWarning: __floordiv__ is deprecated, and its behavior will change in a future version of pytorch. It currently rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values. To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor').
    mkpts0_c = torch.stack([i_ids % data['hw0_c'][1], i_ids // data['hw0_c'][1]], dim=1) * scale0

test/test_contrib.py::TestImageStitcher::test_exception[cpu-float32]
test/feature/test_loftr.py::TestLoFTR::test_mask[cpu]
test/feature/test_loftr.py::TestLoFTR::test_gradcheck[cpu]
  /home/runner/work/kornia/kornia/kornia/feature/loftr/utils/coarse_matching.py:244: UserWarning: __floordiv__ is deprecated, and its behavior will change in a future version of pytorch. It currently rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values. To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor').
    mkpts1_c = torch.stack([j_ids % data['hw1_c'][1], j_ids // data['hw1_c'][1]], dim=1) * scale1

test/augmentation/test_augmentation.py: 14 warnings
test/augmentation/test_backward.py: 162 warnings
  /home/runner/work/kornia/kornia/kornia/augmentation/_2d/intensity/color_jitter.py:85: DeprecationWarning: `ColorJitter` is now following Torchvision implementation. Old behavior can be retrieved by instantiating `ColorJiggle`.
    warnings.warn(

test/augmentation/test_augmentation.py::TestRandomGaussianBlur::test_smoke
test/augmentation/test_augmentation.py::TestRandomGaussianBlur::test_cardinality[cpu-float32-batch_shape0]
test/augmentation/test_augmentation.py::TestRandomGaussianBlur::test_cardinality[cpu-float32-batch_shape1]
test/augmentation/test_augmentation.py::TestRandomGaussianBlur::test_noncontiguous[cpu-float32]
test/augmentation/test_augmentation.py::TestRandomGaussianBlur::test_gradcheck[cpu-float32]
test/augmentation/test_augmentation.py::TestRandomGaussianBlur::test_gradcheck_class_non_deterministic[cpu-float32]
test/augmentation/test_augmentation.py::TestRandomGaussianBlur::test_module[cpu-float32]
  /home/runner/work/kornia/kornia/kornia/augmentation/_2d/intensity/gaussian_blur.py:72: DeprecationWarning: `RandomGaussianBlur` has changed its behavior and now randomly sample sigma for both axes. To retrieve old behavior please consider using kornia.filters.GaussianBlur2d
    warnings.warn(

test/augmentation/test_augmentation.py::TestRandomRGBShift::test_onnx_export[cpu-float32]
  /home/runner/work/kornia/kornia/kornia/augmentation/base.py:109: TracerWarning: torch.tensor results are registered as constants in the trace. You can safely ignore this warning if you use this function to create tensors out of constant variables that would be the same every time you call this function. In any other case, this might cause the trace to be incorrect.
    batch_prob = tensor([True])

test/augmentation/test_augmentation.py::TestRandomRGBShift::test_onnx_export[cpu-float32]
  /home/runner/work/kornia/kornia/kornia/augmentation/base.py:115: TracerWarning: Converting a tensor to a Python number might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
    if batch_prob.sum().item() == 1:

test/augmentation/test_augmentation.py::TestRandomRGBShift::test_onnx_export[cpu-float32]
  /home/runner/work/kornia/kornia/kornia/augmentation/base.py:118: TracerWarning: torch.tensor results are registered as constants in the trace. You can safely ignore this warning if you use this function to create tensors out of constant variables that would be the same every time you call this function. In any other case, this might cause the trace to be incorrect.
    elem_prob = tensor([True] * batch_shape[0])

test/augmentation/test_augmentation.py::TestRandomRGBShift::test_onnx_export[cpu-float32]
  /home/runner/work/kornia/kornia/kornia/augmentation/base.py:147: TracerWarning: Converting a tensor to a Python number might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
    _params = self.generate_parameters(torch.Size((int(to_apply.sum().item()), *batch_shape[1:])))

test/augmentation/test_augmentation.py::TestRandomRGBShift::test_onnx_export[cpu-float32]
  /home/runner/work/kornia/kornia/kornia/augmentation/base.py:153: TracerWarning: torch.tensor results are registered as constants in the trace. You can safely ignore this warning if you use this function to create tensors out of constant variables that would be the same every time you call this function. In any other case, this might cause the trace to be incorrect.
    input_size = tensor(batch_shape, dtype=torch.long)

test/augmentation/test_augmentation.py::TestRandomRGBShift::test_onnx_export[cpu-float32]
  /home/runner/work/kornia/kornia/kornia/augmentation/base.py:250: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
    if not to_apply.any():

test/augmentation/test_augmentation.py::TestRandomRGBShift::test_onnx_export[cpu-float32]
  /home/runner/work/kornia/kornia/kornia/augmentation/base.py:254: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
    elif to_apply.all():

test/augmentation/test_augmentation.py::TestRandomRGBShift::test_onnx_export[cpu-float32]
  /home/runner/work/kornia/kornia/kornia/testing/__init__.py:493: TracerWarning: Converting a tensor to a Python boolean might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
    if len(x.shape) < 3 or x.shape[-3] != 3:

test/core/test_tensor_wrapper.py::TestTensorWrapper::test_unary_ops[cpu-float32]
  /home/runner/work/kornia/kornia/kornia/core/tensor_wrapper.py:103: UserWarning: floor_divide is deprecated, and will be removed in a future version of pytorch. It currently rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values.
  To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor'). (Triggered internally at  /opt/conda/conda-bld/pytorch_1646755832681/work/aten/src/ATen/native/BinaryOps.cpp:607.)
    return wrap(func(*args, **kwargs), cls)

test/enhance/test_histogram.py::TestImageHistogram2d::test_jit[cpu-float32-triangular]
  /usr/share/miniconda/envs/test/lib/python3.10/site-packages/torch/functional.py:568: UserWarning: torch.meshgrid: in an upcoming release, it will be required to pass the indexing argument. (Triggered internally at  /opt/conda/conda-bld/pytorch_1646755832681/work/aten/src/ATen/native/TensorShape.cpp:2228.)
    return _VF.meshgrid(tensors, **kwargs)  # type: ignore[attr-defined]

test/feature/test_affine_shape_estimator.py::TestLAFAffineShapeEstimator::test_shape[cpu]
  /home/runner/work/kornia/kornia/test/feature/test_affine_shape_estimator.py:56: DeprecationWarning: `LAFAffineShapeEstimator` default behaviour is changed and now it does preserve original LAF orientation. Make sure your code accounts for this.
    ori = LAFAffineShapeEstimator().to(device)

test/feature/test_affine_shape_estimator.py::TestLAFAffineShapeEstimator::test_shape_batch[cpu]
  /home/runner/work/kornia/kornia/test/feature/test_affine_shape_estimator.py:63: DeprecationWarning: `LAFAffineShapeEstimator` default behaviour is changed and now it does preserve original LAF orientation. Make sure your code accounts for this.
    ori = LAFAffineShapeEstimator().to(device)

test/feature/test_affine_shape_estimator.py::TestLAFAffineShapeEstimator::test_print[cpu]
  /home/runner/work/kornia/kornia/test/feature/test_affine_shape_estimator.py:68: DeprecationWarning: `LAFAffineShapeEstimator` default behaviour is changed and now it does preserve original LAF orientation. Make sure your code accounts for this.
    sift = LAFAffineShapeEstimator()

test/feature/test_affine_shape_estimator.py::TestLAFAffineShapeEstimator::test_toy_preserve[cpu-float32]
  /home/runner/work/kornia/kornia/test/feature/test_affine_shape_estimator.py:81: DeprecationWarning: `LAFAffineShapeEstimator` default behaviour is changed and now it does preserve original LAF orientation. Make sure your code accounts for this.
    aff = LAFAffineShapeEstimator(32, preserve_orientation=True).to(device, dtype)

test/feature/test_affine_shape_estimator.py::TestLAFAffineShapeEstimator::test_gradcheck[cpu]
  /home/runner/work/kornia/kornia/test/feature/test_affine_shape_estimator.py:104: DeprecationWarning: `LAFAffineShapeEstimator` default behaviour is changed and now it does preserve original LAF orientation. Make sure your code accounts for this.
    assert gradcheck(

test/feature/test_affine_shape_estimator.py::TestLAFAffNetShapeEstimator::test_shape[cpu]
  /home/runner/work/kornia/kornia/test/feature/test_affine_shape_estimator.py:131: DeprecationWarning: `LAFAffNetShapeEstimator` default behaviour is changed and now it does preserve original LAF orientation. Make sure your code accounts for this.
    ori = LAFAffNetShapeEstimator(False).to(device).eval()

test/feature/test_affine_shape_estimator.py::TestLAFAffNetShapeEstimator::test_pretrained[cpu]
  /home/runner/work/kornia/kornia/test/feature/test_affine_shape_estimator.py:138: DeprecationWarning: `LAFAffNetShapeEstimator` default behaviour is changed and now it does preserve original LAF orientation. Make sure your code accounts for this.
    ori = LAFAffNetShapeEstimator(True).to(device).eval()

test/feature/test_affine_shape_estimator.py::TestLAFAffNetShapeEstimator::test_shape_batch[cpu]
  /home/runner/work/kornia/kornia/test/feature/test_affine_shape_estimator.py:145: DeprecationWarning: `LAFAffNetShapeEstimator` default behaviour is changed and now it does preserve original LAF orientation. Make sure your code accounts for this.
    ori = LAFAffNetShapeEstimator().to(device).eval()

test/feature/test_affine_shape_estimator.py::TestLAFAffNetShapeEstimator::test_print[cpu]
  /home/runner/work/kornia/kornia/test/feature/test_affine_shape_estimator.py:150: DeprecationWarning: `LAFAffNetShapeEstimator` default behaviour is changed and now it does preserve original LAF orientation. Make sure your code accounts for this.
    sift = LAFAffNetShapeEstimator()

test/feature/test_affine_shape_estimator.py::TestLAFAffNetShapeEstimator::test_toy[cpu-float32]
  /home/runner/work/kornia/kornia/test/feature/test_affine_shape_estimator.py:154: DeprecationWarning: `LAFAffNetShapeEstimator` default behaviour is changed and now it does preserve original LAF orientation. Make sure your code accounts for this.
    aff = LAFAffNetShapeEstimator(True).to(device, dtype).eval()

test/feature/test_affine_shape_estimator.py::TestLAFAffNetShapeEstimator::test_gradcheck[cpu]
  /home/runner/work/kornia/kornia/test/feature/test_affine_shape_estimator.py:168: DeprecationWarning: `LAFAffNetShapeEstimator` default behaviour is changed and now it does preserve original LAF orientation. Make sure your code accounts for this.
    assert gradcheck(

test/feature/test_integrated.py::TestLAFDescriptor::test_empty[cpu]
  /home/runner/work/kornia/kornia/kornia/feature/integrated.py:44: UserWarning: LAF contains no keypoints torch.Size([1, 0, 2, 3]), returning empty tensor
    warnings.warn(f"LAF contains no keypoints {lafs.shape}, returning empty tensor")

test/feature/test_integrated.py::TestGFTTAffNetHardNet::test_smoke[cpu-float32]
test/feature/test_integrated.py::TestLocalFeatureMatcher::test_nomatch[cpu-float32-loftr_homo]
test/feature/test_integrated.py::TestLocalFeatureMatcher::test_real_gftt[cpu-float32-loftr_homo]
test/tracking/test_planar_tracking.py::TestHomographyTracker::test_smoke[cpu]
test/tracking/test_planar_tracking.py::TestHomographyTracker::test_real[cpu-float32]
  /home/runner/work/kornia/kornia/kornia/feature/integrated.py:184: DeprecationWarning: `LAFAffNetShapeEstimator` default behaviour is changed and now it does preserve original LAF orientation. Make sure your code accounts for this.
    aff_module=LAFAffNetShapeEstimator(True).eval(),

test/filters/test_blur_pool.py::TestMaxBlurPool::test_shape[cpu-float32-True]
test/filters/test_blur_pool.py::TestMaxBlurPool::test_shape_batch[cpu-float32-True]
  /usr/share/miniconda/envs/test/lib/python3.10/site-packages/torch/nn/functional.py:780: UserWarning: Note that order of the arguments: ceil_mode and return_indices will changeto match the args list in nn.MaxPool2d in a future release.
    warnings.warn("Note that order of the arguments: ceil_mode and return_indices will change"

test/geometry/test_homography.py::TestFindHomographyFromLinesDLT::test_nocrash[cpu-float32]
  /home/runner/work/kornia/kornia/kornia/geometry/homography.py:308: RuntimeWarning: SVD did not converge
    warnings.warn('SVD did not converge', RuntimeWarning)

test/geometry/transform/test_imgwarp.py::TestRemap::test_different_size[cpu-float32]
  /usr/share/miniconda/envs/test/lib/python3.10/site-packages/torch/nn/functional.py:4193: UserWarning: Default grid_sample and affine_grid behavior has changed to align_corners=False since 1.3.0. Please specify align_corners=True if the old behavior is desired. See the documentation of grid_sample for details.
</details>

You can check too at the [scheduled action](https://github.com/kornia/kornia/actions/workflows/scheduled_test_cpu.yml)
